blackjack/actions.py

```diff
@@ -35,42 +38,42 @@ def stick_or_twist(hand: Hand, deck: Deck) -> None:
     while True:
         move = input("Stick or twist? Input s or t: ")
 
-        if move == "t":
+        if move.lower() == "t":
             twist(hand, deck)
-        elif move == "s":
+        elif move.lower() == "s":
             print("Player has chosen to stick.")
             game_config.playing = False
         else:
             print_invalid_input(["s", "t"])
         break
 
 
-def reveal_cards(my_hand: Hand, reveal_all=True) -> None:
+def reveal_cards(player_hand: Hand, reveal_all: Boolean = True) -> None:
```

Comment:
Why from xmlrpc.client import Boolean and not built in bool?

Reply:
Oops! This was an accident - nice catch.